### DISCUSSION TITLE
Workbooks in S3 compatible object stores

### DISCUSSION THREAD

**Discussion by D3SL** [JanMarvin/openxlsx2#1059] [Q&A]
Is it possible to use `wb_load()` with an S3 compatible object store? The two main methods of this I can think of are getting the body of a workbook as hex and basically reconstructing it in memory (or temp), or opening a connection to the file directly in the S3 bucket and operating on it in-situ at the far end.

**Comment by JanMarvin** [comment]
Not sure what you mean. We have the wbWorkbook object which is an R6 object. It contains the entire content from the workbook minus some binary blobs which are stored in the temp folder.
Shifting parts of workbooks around is not a trivial task and part of the reason why after more than two years there are still commits to this project
